lib/rubocop/cop/performance/hash_each_methods.rb

```diff
@@ -55,11 +67,11 @@ def autocorrect(node)
           caller, first_method = *receiver
 
           lambda do |corrector|
-            if receiver.method?(:hash)
-              correct_hash(node, corrector)
-            else
+            if (first_method == :keys) || (first_method == :values)
```

Comment:
Should be no need for parentheses here. (False negative of `Style/RedundantParentheses`? 🤔 )